### PR TITLE
update host because CORS

### PR DESCRIPTION
**Pull request by gvonkoss**
Fixes https://jira.ft.com/browse/NFT-910 which was pointing at next-errors.ft.com/search... which wasn't anything. 

cc/ @JakeChampion @i-like-robots

### CODE REVIEW

**Comment by wheresrhys** [comment]
I'll be merging https://github.com/Financial-Times/n-ui/pull/1043 soon. Could you port this fix to https://github.com/Financial-Times/n-topic-search too please

I also wonder if a better fix would be to rethink error pages a bit, but that's just one to chalk up for later

**Comment by JakeChampion** [comment]
@gvonkoss It looks like Circle wants the file to be smaller :/
>FAIL  ./public/n-ui/es5.js: 56.13kB > maxSize 56kB gzip

**Comment by gvonkoss** [comment]
yeah but I don't know how to change that. And I don't know if the n-express warning has anything to do with it

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
   "bundlesize": [
     {
       "path": "./public/n-ui/es5.js",
-      "threshold": "56 Kb"
+      "threshold": "56.5 Kb"
```
Comment (JakeChampion) [inline package.json]: 😂

Comment (gvonkoss) [reply package.json]: 😬

**Pull request merged**